user/contactmethod/store.go

```diff
@@ -96,6 +105,33 @@ func NewDB(ctx context.Context, db *sql.DB) (*DB, error) {
 	}, p.Err
 }
 
+func (db *DB) EnableByValue(ctx context.Context, t Type, v string) error {
+	err := permission.LimitCheckAny(ctx, permission.System)
+	if err != nil {
+		return err
+	}
+
+	c := ContactMethod{Name: "Enable", Type: t, Value: v}
+	n, err := c.Normalize()
+	if err != nil {
+		return err
+	}
+
+	var id string
+	err = db.enable.QueryRowContext(ctx, n.Type, n.Value).Scan(&id)
+
+	if err == nil {
+		// NOTE: maintain a record of consent/dissent
+		logCtx := log.WithFields(ctx, log.Fields{
+			"contactMethodID": id,
+		})
+
+		log.Logf(logCtx, "Contact method resubscribe code received.")
```

Comment:
As with below, this can be `Contact method START received.` to match `STOP`.

We could use "subscribe/resubscribe/unsubscribe", "opt-in/opt-out", etc.. but since STOP is pretty common verbiage, I think START and STOP might be the most predictable/guessable words if one needed to search logs/records for a timestamp.

In this case, I think an explicit predictable keyword is best (even if less readable) since that's the primary reason/use for adding these two log lines.

Reply:
That works, though we will have to be extra cognizant in the future if we are debugging cases using the non-recommended codes as it could lead to confusion while searching the logs. Future/new developers might not know this either. Opt-out/in might be the most appropriate words on second though, since re/un-subscribe could also be confused for keywords.

Reply:
We did go with `opt-out` instead of `STOP` in the previous PR that we merged, so I think it may be best to stick with that path as we move forward.